### PR TITLE
Prove Scenario 90 admin authz fidelity

### PR DESCRIPTION
**Pull request by intel352**
## Summary
- Turns Scenario 90 into a runnable Workflow app + admin example protected by authn/authz, with no committed seed secret and with authz metadata endpoints authorization-gated.
- Uses real admin/auth/authz plugin contribution steps for the admin nav and marks unsupported ABAC/ReBAC modes as unavailable instead of pretending routes exist.
- Adds shell and Playwright coverage for admin navigation, auth iframe bridge, RBAC write behavior, direct-route protection, delegated admin usability, and plugin-surface authorization.

## Verification
- `wfctl validate --skip-unknown-types scenarios/90-admin-tailnet-demo/config/app.yaml`
- `PLUGIN_ADMIN_REPO=/Users/jon/workspace/workflow-plugin-admin/.worktrees/admin-shell-auth-gate PLUGIN_AUTHZ_UI_REPO=/Users/jon/workspace/workflow-plugin-authz-ui/.worktrees/authz-ui-admin-bridge ./test/run.sh` -> `PASS=96 FAIL=0`
- `PLAYWRIGHT_PREFIX=/tmp/scenario90-playwright-codex ./scenarios/90-admin-tailnet-demo/test/run-playwright.sh` -> `1 passed`

## Related
- Filed workflow validator follow-up for YAML boolean edge case: https://github.com/GoCodeAlone/workflow/issues/809

### CODE REVIEW

File: scenarios/90-admin-tailnet-demo/test/run-playwright.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+PLAYWRIGHT_PREFIX="${PLAYWRIGHT_PREFIX:-/tmp/scenario90-playwright-cli}"
+
+if [ ! -x "$PLAYWRIGHT_PREFIX/node_modules/.bin/playwright" ]; then
+  npm --prefix "$PLAYWRIGHT_PREFIX" install --silent @playwright/test
+fi
```
[the remark this answers is not in the document]
Comment (intel352) [reply scenarios/90-admin-tailnet-demo/test/run-playwright.sh:11]: Addressed in daf7f1c: run-playwright.sh now creates PLAYWRIGHT_PREFIX before invoking npm/npx.

File: scenarios/90-admin-tailnet-demo/test/playwright/admin-surfaces.spec.cjs [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  await frame.getByLabel('User', { exact: true }).selectOption('app-user@tailnet');
+  await frame.getByLabel('Role', { exact: true }).selectOption('support');
+  await frame.locator('form.rule-form select').nth(2).selectOption('admin');
+  await frame.locator('label.scope-option').filter({ hasText: 'admin:authz.roles:read' }).locator('input').check();
+  await frame.getByRole('button', { name: 'Add Role' }).click();
+  await expect(frame.locator('body')).toContainText('admin:authz.roles:read');
+  const adminToken = await page.evaluate(() => window.localStorage.getItem('workflow.admin.token'));
+  const rolesAfterWrite = await request.get('/api/authz/roles', {
+    headers: { Authorization: `Bearer ${adminToken}` },
+  });
```
[the remark this answers is not in the document]
Comment (intel352) [reply scenarios/90-admin-tailnet-demo/test/playwright/admin-surfaces.spec.cjs]: Addressed in daf7f1c: the Playwright test now reads data-token-storage-key from the admin shell before loading the token from localStorage.

File: scenarios/90-admin-tailnet-demo/seed/seed.sh [hunk 3]
```diff
@@ -128,7 +136,17 @@ build_plugin() {
 BUILD_DIR="$SCENARIO_DIR/.build"
 rm -rf "$BUILD_DIR"
 mkdir -p "$BUILD_DIR/plugins" "$BUILD_DIR/admin-ui" "$BUILD_DIR/authz-ui" "$BUILD_DIR/app-ui" "$BUILD_DIR/data"
+if [ -z "${SCENARIO90_SEED_TOKEN:-}" ]; then
+  if command -v openssl >/dev/null 2>&1; then
+    SCENARIO90_SEED_TOKEN="$(openssl rand -hex 32)"
+  else
+    SCENARIO90_SEED_TOKEN="$(uuidgen | tr '[:upper:]' '[:lower:]')-$(date +%s)"
+  fi
+fi
```
[the remark this answers is not in the document]
Comment (intel352) [reply scenarios/90-admin-tailnet-demo/seed/seed.sh:149]: Addressed in daf7f1c: seed token generation now checks uuidgen and falls back through od before a final timestamp/PID fallback.

File: scenarios/90-admin-tailnet-demo/docker-compose.yml [hunk 4]
```diff
@@ -6,6 +6,7 @@ services:
     environment:
       WFCTL_PLUGIN_DIR: /data/plugins
       JWT_SECRET: scenario-90-local-demo-secret-32-bytes-minimum
+      SCENARIO90_SEED_TOKEN: ${SCENARIO90_SEED_TOKEN:?SCENARIO90_SEED_TOKEN required}
     ports:
```
[the remark this answers is not in the document]
Comment (intel352) [reply scenarios/90-admin-tailnet-demo/docker-compose.yml:9]: Addressed in daf7f1c: docker-compose no longer requires SCENARIO90_SEED_TOKEN because seed.sh substitutes the per-run token into the generated .build/app.yaml before compose starts.
